### DISCUSSION TITLE
is this safe

### DISCUSSION THREAD

**Discussion by hsgdhehdhdh** [CaptainChicky/Trojan.JS.Youareanidiot#3] [Q&A] [answered]
I went online and found out that Trojan.JS.Youareanidiot spams pop-ups until you get BSOD

**Comment by CaptainChicky** [comment, marked as answer]
> is this safe

I know this is late asf, but to answer your question, sort of?

What is happenning is basically it's a browser fork bomb that spams windows until your computer runs out of resources. You won't get a BSOD, but you'll get lagged to oblivion, without any practical way of disabling it in time, if you are caught off guard. At least if you run it on IE.

You can simply restart and all will be well. Again, please read the readme, which links you to danooct1's video on this. 

Regardless, the fork bomb doesn't really work on modern browsers, which

1. Have better security, which disables the ability for capturing shortcuts to work, at least for this trojan.
2. Block popups by default, and also block too many popups, even if you allow popups.
3. Don't run flash, so you don't hear the audio spam.

In other words, if you run this on a modern browser, simply ctrl+w ing the tabs would work. If you run this on IE, a simple trip to the task manager or a taskkill command on the terminal is all it takes to stop it as well.
